src/App.jsx

```
@@ -6,9 +6,13 @@ import "./App.css";
 function App() {
     const [count, setCount] = useState(0);
 
+    if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)